flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/data/AbstractTestFlinkAvroReaderWriter.java

```diff
@@ -65,12 +63,13 @@ public class TestFlinkAvroReaderWriter extends DataTest {
           Types.NestedField.optional(9, "decimal", Types.DecimalType.of(4, 2)));
 
   @Override
-  protected void writeAndValidate(Schema schema) throws IOException {
-    List<Record> expectedRecords = RandomGenericData.generate(schema, NUM_RECORDS, 1991L);
-    writeAndValidate(schema, expectedRecords, NUM_RECORDS);
-  }
+  protected abstract void writeAndValidate(Schema schema) throws IOException;
 
-  private void writeAndValidate(Schema schema, List<Record> expectedRecords, int numRecord)
+  void writeAndValidate(
+      Schema schema,
+      List<org.apache.iceberg.data.Record> expectedRecords,
```

Comment:
Does it need to be fully qualified?

Reply:
Afair, the reason is to avoid confusion with another `Record`. Let me check.